aws/validators.go

```diff
@@ -1395,6 +1395,43 @@ func validateAwsKmsName(v interface{}, k string) (ws []string, es []error) {
 	return
 }
 
+func validateAwsKmsGrantOperation(v interface{}, k string) (ws []string, es []error) {
+	value := v.(string)
+
+	validTypes := map[string]bool{
+		"Decrypt":                         true,
+		"Encrypt":                         true,
+		"GenerateDataKey":                 true,
+		"GenerateDataKeyWithoutPlaintext": true,
+		"ReEncryptFrom":                   true,
+		"ReEncryptTo":                     true,
+		"CreateGrant":                     true,
+		"RetireGrant":                     true,
+		"DescribeKey":                     true,
+	}
+
+	if _, ok := validTypes[value]; !ok {
+		es = append(es, fmt.Errorf(
+			"%s must be one of the following: [ Decrypt, Encrypt, GenerateDataKey, GenerateDataKeyWithoutPlaintext, ReEncryptFrom, ReEncryptTo, CreateGrant, RetireGrant, DescribeKey ]", k))
```

Comment:
Could we programmatically generate this list, to prevent drift?

Reply:
Will pull from the above map above thanks. By the way this kind of duplication is pretty common in this file, is probably worth cleaning up.

Reply:
lets follow the suggestion in this PR, your suggestion is welcome but lets focus the conversation on the PR it self 

alvaro

Reply:
No problem, was just bringing a small possible issue to your attention.